modules/common/math/box2d.h

```diff
@@ -236,15 +238,29 @@ class Box2d {
    */
   std::string DebugString() const;
 
+  void InitCorners();
+
+  double max_x() const { return max_x_; }
+  double min_x() const { return min_x_; }
+  double max_y() const { return max_y_; }
+  double min_y() const { return min_y_; }
+
  private:
   Vec2d center_;
   double length_ = 0.0;
   double width_ = 0.0;
   double half_length_ = 0.0;
   double half_width_ = 0.0;
-  double heading_ = 0.0;
+  double heading_ = 1.0;
```

Comment:
why change default value to 1.0? 0.0 may help to indicate that it is not initialized.